README.md

```diff
@@ -13,19 +13,17 @@ This repository includes files with plain SQL that can be used to recreate a dat
 - Check [queries.sql](./queries.sql) for examples of queries that can be run on a newly created database. **Important note: this file might include queries that make changes in the database (e.g., remove records). Use them responsibly!**
 
```

Comment:
Please add a project title and description to the README. This way it looks more professional.